src/tests/builtins.pl

```diff
@@ -125,6 +125,7 @@
     \+ \+ (g(B) = B, g(A) = A, variant(A, B)),
     keysort([1-1,1-1],[1-1,1-1]),
     \+ \+ findall(Sorted, keysort([2-99,1-a,3-f(_),1-z,1-a,2-44],Sorted), [[1-a,1-z,1-a,2-99,2-44,3-f(_)]]),
-    \+ \+ findall(X, keysort([X-1,1-1],[2-1,1-1]), [2]).
+    \+ \+ findall(X, keysort([X-1,1-1],[2-1,1-1]), [2]),
+    2 is popcount(5).
```

Comment:
It would be better to use `(=:=)/2` here since both sides are ground: `2 =:= popcount(5)`.